inspirehep/config.py

```diff
@@ -672,3 +672,25 @@ def _(x):
 # Submission
 # ==========
 CFG_ROBOTUPLOAD_SUBMISSION_BASEURL = "http://localhost:5000"
+
+# Inspire subject translation
+# ===========================
+LITERATURE_CATEGORIES = {
```

Comment:
This is actually a subset of: https://github.com/inspirehep/inspire/blob/master/bibconvert/arxiv-to-inspire-categories.kb

Can you use that source?